models/dvmdostem/R/model2netcdf.dvmdostem.R

```diff
@@ -0,0 +1,76 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2016 NCSA.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the 
+# University of Illinois/NCSA Open Source License
+# which accompanies this distribution, and is available at
+# http://opensource.ncsa.illinois.edu/license.html
+#-------------------------------------------------------------------------------
+
+##' @name model2netcdf.dvmdostem
+##' @title Code to convert dvmdostem netcdf output into into CF standard
+##'
+##' @param outdir Location of dvmdostem model output
+##' 
+##' @examples  
+##' \dontrun{
+##' # example code here?
+##' }
+##' 
+##' @export
+##'
+##' @author Tobey Carman, Shawn Serbin
+##' @importFrom ncdf4 ncdim_def ncvar_def ncatt_get ncvar_add
+model2netcdf.dvmdostem <- function(outdir) {
+
+  # Define PEcAn style dimensions
+  lond <- ncdf4::ncdim_def(name='lon',
+                           units="degrees_east",
+                           vals=c(1), # <=== read from dvmdostem file!
```

Comment:
If these values have not yet been read in, but will be read from the model's output file, then you should just wait to define the dimensions until you have the data in hand so that you don't have to figure out how to change it to the correct values later.